driver/src/main/scala/api/commands/writeresult.scala

```diff
@@ -32,8 +32,7 @@ sealed trait WriteResult {
 object WriteResult {
   def lastError(result: WriteResult): Option[LastError] = result match {
```

Comment:
The comment corresponds to the `insert` use case, not bulk.

Would be nice to ensure there is a test checking all the error details in case of bulk.

Then the question is whether it should really fails, whereas MongoDB indicates `ok`.